### ISSUE TITLE
Refering to static property's within static property's of a React Class gives error

### ISSUE THREAD

**Issue by jamaldewitte** [CompuIves/codesandbox-client#373]
Hi, upon using codesandbox I found the following issue: 

Code: 

```
class Example extends React.Component {
  static hello = "__hello__"
 static hi = Example.hello

render() {
return <div> Example </div> }
}
```

Expected:`  static hi `to be equal to `'__hello__'`
Result: `_TypeError. Cannot read property 'hello' of undefined`

Kind regards,

Jamal

**Comment by lbogdan** [comment]
Hey @jamaldewitte , thanks for your report, and sorry for the "small" delay! :slightly_smiling_face: 

So, this is an interesting one. I could trace it back to ["Class properties work differently with transform-decorators-legacy"](https://github.com/loganfsmyth/babel-plugin-transform-decorators-legacy/issues/17), and it's caused by [`transform-decorators-legacy` coming before `transform-class-properties`](https://github.com/CompuIves/codesandbox-client/blob/master/packages/app/src/sandbox/eval/transpilers/babel/babel-parser.js#L8-L9). That was changed in #154 to fix #151 , and it's also recommended by [`transform-decorators-legacy` documentation](https://github.com/loganfsmyth/babel-plugin-transform-decorators-legacy#note-order-of-plugins-matters). As [the issue](https://github.com/loganfsmyth/babel-plugin-transform-decorators-legacy/issues/17) is still opened, I don't think we currently have a solution for this.

@CompuIves Any idea?

**Comment by CompuIves** [comment]
This is a hard one. Our `new` template has one difference with CRA, which is that we support decorators. We released with this support and I realized this too late to remove it from the template. As I understand this is the cause of the error, having decorator support and class transform, right? 

If that's the case there's not much that we can do currently except for waiting for either:

1. Create React App v2
2. A fix in the issue mentioned by @lbogdan 

A third option would be to remove decorators from `new`, but this would break existing sandboxes relying on it.
